fv/test-connection/test-connection.go

```diff
@@ -35,11 +36,12 @@ import (
 const usage = `test-connection: test connection to some target, for Felix FV testing.
 
 Usage:
-  test-connection <namespace-path> <ip-address> <port> [--source-port=<source>] [--protocol=<protocol>]
+  test-connection <namespace-path> <ip-address> <port> [--source-port=<source>] [--protocol=<protocol>] [--loop-with-file=<file>]
 
 Options:
   --source-port=<source>  Source port to use for the connection [default: 0].
-  --protocol=<protocol>  Protocol to test [default: tcp].
+  --protocol=<protocol>   Protocol to test [default: tcp].
+  --loop-with-file=<file>  Whether to send messages repeatedly, file is used for synchronization
```

Comment:
I found it a bit fiddly to unpick the loop-with-file "protocol".  Maybe you could spell it out in a comment?  I think
- File should exist before this binary is started.
- After first successful connection, this binary deletes the file.
- If the file reappears, this binary shuts down.

Reply:
Wrote a comment about it.